tools/sds-aspect-model-maven-plugin/src/main/java/io/openmanufacturing/sds/aspectmodel/Validate.java

```diff
@@ -39,9 +41,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
       final Set<Try<VersionedModel>> resolvedModels = loadAndResolveModels();
       for ( final Try<VersionedModel> versionedModel : resolvedModels ) {
-         final ValidationReport report = validator.validate( versionedModel );
-         if ( !report.conforms() ) {
-            throw new MojoFailureException( report.toString() );
+         final List<Violation> violations = validator.validateModel( versionedModel );
+         if ( !violations.isEmpty() ) {
+            throw new MojoFailureException( new ViolationFormatter().apply( violations ) );
```

Comment:
IMHO the option to print the detailed output for any possible violations should be available here as well.

Reply:
Done